.travis.yml

```diff
@@ -7,9 +7,12 @@ language: objective-c
 env:
   matrix:
     
-    - CONDA_PY=27  
-    - CONDA_PY=34  
-    - CONDA_PY=35  
+    - CONDA_NPY=110  CONDA_PY=27
```

Comment:
Yeah, I tried, but it was complaining about missing CI tokens. As it was quite late and I wanted to get something out to fix a problem that someone was having when using `h5py`, I picked the quickly and dirty route of making the change manually. Maybe we can chat about the issue I was running into and see if there was a usage error on my part or something that needs fixing.